[store] 2024.10.04 Release Notes (#9440)
organicmaps/organicmaps: android/app/src/fdroid/play/listings/en-US/release-notes.txt

> - Track Metadata Editing: Modify the name, color, category, and description of tracks, or delete them.

Reviewer: ```suggestion
- Edit Tracks: Change name, color, list and description of tracks, or delete them.
```

> - Open in Another App: Open saved points from Organic Maps in apps like taxi, delivery, transport, or other map apps.

Reviewer: ```suggestion
- Open in Another App: Select any map point in Organic Maps and open in apps like taxi, delivery, transport, or other map apps.
```

> - Map Data: Updated as of October 1, now including isolines for Egypt, Adygea, Dagestan.

Reviewer: ```suggestion
- Map Data: Updated as of October 1, now with added isolines for Egypt and improved for Adygea, Dagestan.
```
